src/shared.py

```diff
@@ -1,21 +1,25 @@
-﻿from __future__ import division
+﻿"""
+src/shared.py
+=============
+"""
```

Comment:
Some shared functions
Should be moved to different places and this file removed, but it needs refactoring

Reply:
```
Some shared functions

.. deprecated:: 0.6.3
  Should be moved to different places and this file removed,
  but it needs refactoring.
```